dockstore-webservice/src/main/java/io/dockstore/webservice/resources/SourceControlResourceInterface.java

```diff
@@ -45,18 +45,20 @@
     /**
      * Refreshes user's Bitbucket token.
      *
-     * @param token
+     * @param bitbucketToken
      * @param client
      * @param tokenDAO
      * @param bitbucketClientID
      * @param bitbucketClientSecret
      */
-    default void refreshBitbucketToken(Token token, HttpClient client, TokenDAO tokenDAO, String bitbucketClientID,
+    default void refreshBitbucketToken(Token bitbucketToken, HttpClient client, TokenDAO tokenDAO, String bitbucketClientID,
         String bitbucketClientSecret) {
 
+        LOG.info("Refreshing the BitBucket Token");
         String refreshUrl = BITBUCKET_URL + "site/oauth2/access_token";
-        String payload = "grant_type=refresh_token&refresh_token=" + token.getRefreshToken();
-        refreshToken(refreshUrl, token, client, tokenDAO, bitbucketClientID, bitbucketClientSecret, payload);
+        String payload = "client_id=" + bitbucketClientID + "&client_secret=" + bitbucketClientSecret
```

Comment:
There is the risk that some value needs to be url encoded. But the risk was already there, so I guess so far, or maybe never, there are no values that have required encoding.